### ISSUE TITLE
Question: is it possible to support server-side validation with the form component?

### ISSUE THREAD

**Issue by Riari** [ElemeFE/element#1216]
Hi,

This is a really nice library. Thanks for sharing and for the English translation.

I want to use this: http://element.eleme.io/#/en-US/component/form or the standalone input component with the ability to display validation errors that come from the server.

I can do it by simply displaying an error below the component like this:

```
<el-input
  placeholder="Username or email address"
  v-model="identity"
>
</el-input>
<error>{{ errors.username }}</error>
```

But that leaves me without a clean, simple way of modifying the input's style (i.e. red border, like the built in form validation does) and I'd prefer to do something like this:

```
<el-input
  placeholder="Username or email address"
  v-model="identity"
  :error="errors.username"
>
</el-input>
```

...which would display the error message if it's defined, as well as change the style accordingly.

I tried wrapping the component in a custom one to achieve that but had difficulty syncing the input value. Any recommendations?

Edit: I think this might be related to #983

**Comment by Leopoldthecoder** [comment]
Fixed in #1221

**Comment by Riari** [comment]
Awesome. Thanks!
